test/services/ldap-config/service.integration.test.js

```diff
@@ -261,7 +260,11 @@ describe('LdapConfigService', () => {
 				.set('content-type', 'application/json');
 			const response = await request.send(knownBadConfig);
 			expect(response.status).to.equal(400);
-			expect(response.body.message).to.include("request.body should have required property 'rootPath'");
+			expect(response.body.validation_errors).to.deep.include({
```

Comment:
validation_errors is the key for all validation errors? If it is used in error middleware for mapping?

Reply:
Yes, it is a key for validation error details - if needed. Please have a look at https://docs.hpi-schul-cloud.org/display/TSC/Error+Handling+Guildelines#ErrorHandlingGuildelines-ErrorsintheServer